cpp/cmake/thirdparty/get_raft.cmake

```diff
@@ -81,13 +81,13 @@ endfunction()
 # To use a different RAFT locally, set the CMake variable
 # CPM_raft_SOURCE=/path/to/local/raft
 find_and_configure_raft(VERSION          ${CUML_MIN_VERSION_raft}
-                        FORK             rapidsai
-                        PINNED_TAG       branch-${CUML_BRANCH_VERSION_raft}
+                        FORK             cjnolet
+                        PINNED_TAG       braft-22.12-test
```

Comment:
Can this be switched back to standard values, or is there an associated RAFT PR?

Reply:
That's just a fork of branch-22.12 to test an issue w/ pinning raft commits in cuml.

Reply:
Got it! So we just need to switch it back before merge, but this demonstrates the functionality, right?